package.json

```diff
@@ -12,7 +12,7 @@
     "cors-anywhere": "^0.4.1",
     "jsonwebtoken": "^8.4.0",
     "lodash": "^4.17.11",
-    "material-ui": "^0.20.2",
+    "material-ui": "^1.0.0-beta.47",
```

Comment:
Oh, I hadn't noticed this one... Your dependency is badly outdated for this version. Upgrading to a more recent version of material-ui will require some work, you'd better not delay that task too much.

Reply:
Updated